problem4/problem4.js

```diff
@@ -5,10 +5,12 @@ var fahrenheitValue = document.getElementById('fahrenheit-value');
 // ==== Implement your code here! ====
 function convert(celsiusValue) {
     console.log(celsiusValue);
-
-    fahrenheitValue.innerHTML = 'Your answer goes here';
+    
+    fahrenheitValue.innerHTML = (celsiusValue*9/5)+32;
```

Comment:
What happens when the user enters a text value in the input field? Let's add code to check for that case and display an error message (like, Entered value is not a number).